apps/website/.vuepress/theme/components/Home.vue

```diff
@@ -107,15 +107,25 @@
       <h3 cds-text="display center medium" cds-layout="m-b:xl">Popular Resources</h3>
       <div cds-layout="grid gap:lg cols@xs:12 cols@sm:6">
         <div class="image-cell" cds-layout="m-b:sm m-b@sm:xxl">
-          <img cds-layout="container:fill" src="/images/home/img-pr-components.svg" aria-hidden="true" />
-          <h4 cds-text="title medium" cds-layout="m-t:lg">Components</h4>
+          <a href="/components/accordion/">
+            <img cds-layout="container:fill" src="/images/home/img-pr-components.svg" aria-hidden="true" />
+          </a>
+          <a href="/components/accordion/">
```

Comment:
Are we linking to accordion because there isn't a component overview page yet?

Reply:
Yes, I was told that creating an introduction page for the components section is on our roadmap. Once we have that page, we will use that page's url for this link.